### ISSUE TITLE
kotlin value class "get property" is always showing as not called. 

### ISSUE THREAD

**Issue by tafypz** [Kotlin/kotlinx-kover#302]
**Describe the bug**
value classes "getters" are never shown as covered when they are used in test cases
this happens in both jacoco and intellij  engines.

**Errors**
coverage shows 0% coverage

**Expected behavior**
coverage showing 100% when used.

**Reproducer**
create a value class:
@JvmInline
value class MyValueClass(val data: String)

in junit 5 test class:
@Test 
fun myTest() {
  val v = MyValueClass("ABC")
  if(v.data != "ABC")
     error("not abc")
}
 

**Reports**
If applicable, report files or screenshots.

**Environment**
 - Kover Gradle Plugin version: 0.6.2
 - Gradle version: 7.6
 - Kotlin project type: jvm

**Comment by zuevmaxim** [comment]
@tafypz Please try a fix with `1.0.707` engine version:
for Kotlin Gradle script
```
kover {
    engine.set(kotlinx.kover.api.IntellijEngine("1.0.707"))
}
```
for Groovy
```
kover {
    engine = new kotlinx.kover.api.IntellijEngine("1.0.707")
}
```

**Comment by tafypz** [comment]
@zuevmaxi,m It did work with that version (well technically I see that the value class is omitted from the report)
Is that what is to be expected ?

**Comment by zuevmaxim** [comment]
@tafypz , yes. Unfortunately, we cannot say for sure whether getter and constructor have been called or not, so we exclude this lines. If such a class has some more methods, they will not be omitted

**Comment by tafypz** [comment]
@zuevmaxim it looks like if the value class is formatted that way, it gets included in the report:
```
@JvmInline
 value class EntityFindByIdCommandParameter(
     val entityId: EntityId
 )
```

however formatted that way, it is omitted:

```
@JvmInline
value class EntityFindByIdCommandParameter(val entityId: EntityId)
```

**Comment by zuevmaxim** [comment]
Thanks, looks like we should take care of the constructor too

**Comment by zuevmaxim** [comment]
@tafypz Please try a fix with `1.0.708` engine version
Now only getter is ignored, but all the rest lines are covered as expected

**Comment by tafypz** [comment]
@zuevmaxim 1.0.708 works as I expected.
Thanks !
